### PR TITLE
feat(docs): add the a11y section for Button component

### LINKED ISSUE #1561: Add accessibility section for `button`
### Subject

accessibility section for `button`

#### Create an accessibility section using the link below
https://yamada-ui.com/components/disclosure/tabs/accessibility#aria-roles-and-attributes
https://v2.chakra-ui.com/docs/components/button#accessibility

#### Include the following content
* aria, role, tabindex property are supported
* link for W3C(https://www.w3.org/WAI/ARIA/apg/patterns/button/)

#### reference
https://github.com/yamada-ui/yamada-ui/issues/1737

### Are you willing to participate in creating this documentation page and create a pull request?

None

### PR DESCRIPTION
**Pull request by taiseiiiii**
Closes #1561

### in English

スクリーンショット 2024-10-16 13 02 00（3）

### in Japanese

スクリーンショット 2024-10-16 13 03 17（3）

### CODE REVIEW

**Comment by taiseiiiii** [comment]
@108yen 
I intend to demonstrate that this component does not need `role="button"`
because it already has a default role of button.
reference
https://stackoverflow.com/questions/26934719/do-i-need-role-button-on-a-button
https://github.com/yamada-ui/yamada-ui/pull/2298#issuecomment-2235170461

I included this because there was a similar description in the link below
https://v2.chakra-ui.com/docs/components/button#accessibility

Should I remove this phrase and the section "ARIA Roles and Attributes"?

**Comment by 108yen** [comment]
> @108yen I intend to demonstrate that this component does not need `role="button"` because it already has a default role of button. reference https://stackoverflow.com/questions/26934719/do-i-need-role-button-on-a-button [#2298 (comment)](https://github.com/yamada-ui/yamada-ui/pull/2298#issuecomment-2235170461)
> 
> I included this because there was a similar description in the link below https://v2.chakra-ui.com/docs/components/button#accessibility
> 
> Should I remove this phrase and the section "ARIA Roles and Attributes"?

@taiseiiiii 
I know that.
I made a comment regarding the format of the writing.
Match the writing style with other components.

**Comment by taiseiiiii** [comment]
@108yen 
I understand. Thanks for your point.
Would it be possible to confirm modifications?

**Review by 108yen — DISMISSED** [review]

File: docs/contents/components/forms/button/accessibility.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tab: Accessibility
+---
+
+The `Button` follows the [WAI-ARIA - Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/) for accessibility.
+
+## Keyboard Navigation
+
+| Key                    | Description                          | State |
+| ---------------------- | ------------------------------------ | ----- |
+| `Space`, <br/> `Enter` | When Button has focus, activates it. | -     |
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/button/accessibility.mdx:16]: Please remove `<br />` element.

File: docs/contents/components/forms/button/accessibility.ja.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+tab: アクセシビリティ
+---
+
+`Button`は、アクセシビリティに関して[WAI-ARIA - Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/button/)に従います。
+
+## キーボード操作
+
+| キー                   | 説明                                     | 状態 |
+| ---------------------- | ---------------------------------------- | ---- |
+| `Space`, <br/> `Enter` | フォーカスしているボタンを作動させます。 | -    |
```
Comment (hirotomoyamada) [inline docs/contents/components/forms/button/accessibility.ja.mdx:16]: Please remove `<br />` element.

Comment (hirotomoyamada) [inline docs/contents/components/forms/button/accessibility.ja.mdx:16]: Please change "フォーカスしているボタンを作動させます。" to "フォーカスしているボタンをアクティブにします。".

File: docs/contents/components/forms/button/accessibility.mdx (hunk 1 above)
Comment (hirotomoyamada) [reply docs/contents/components/forms/button/accessibility.mdx:16]: Change "Button" to "element".

**Comment by taiseiiiii** [comment]
@hirotomoyamada 
Thanks for your feedback.
I have addressed your comments. Please review.

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**